presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClientS3.java

```diff
@@ -457,10 +458,10 @@ public void updateTableLocation(String databaseName, String tableName, String lo
                 }
                 table.get().getSd().setLocation(location);
                 try (HiveMetastoreClient client = clientProvider.createMetastoreClient()) {
-                    client.alter_table(databaseName, tableName, table.get());
+                    client.alterTable(databaseName, tableName, table.get());
                 }
             }
-            catch (TException e) {
+            catch (IOException e) {
```

Comment:
This is because the ```HiveMetastoreClient``` interface extends ```Closable``` and close throws ```IOException``` and since we are calling an instance of the ```HiveMetastoreClient``` it will throw. I can add close to the interface without the exception so these don't have to be caught.